src/onyx/messaging/aeron/peer_manager.clj

```diff
@@ -36,7 +35,7 @@
           (while (.hasNext iterator)
             (let [kv ^Int2ObjectHashMap$EntryIterator (.next iterator)
                   k ^java.lang.Integer (.getKey kv) 
-                  v ^PeerChannels (.getValue kv)]
```

Comment:
Yes, the peer manager may now have different objects types in it. I may revise this fact though.